.github/workflows/Tests.yaml

```diff
@@ -531,13 +507,10 @@ jobs:
           -D CHARM_ROOT=/work/charm_6_10_2/multicore-linux-x86_64-${CHARM_CC}
           -D CMAKE_BUILD_TYPE=Debug
```

Comment:
Can arch-related issues occur in `Release` mode but not in `Debug`?

Reply:
I suppose it might be possible in principle, but I don't believe that Blaze does any of its vectorization any differently for debug builds.